frontend/src/classes/SpatialChat.ts

```diff
@@ -0,0 +1,47 @@
+import Player, { ServerPlayer } from "./Player";
+
+
+export interface ServerChatEntry {
```

Comment:
This is from the backend branch. I've basically copied their structure into this here.